doc/api/stream.md

```diff
@@ -1448,8 +1448,8 @@ added: v8.0.0
   argument) when you are done writing any remaining data.
 
 Note: `_final()` **must not** be called directly.  It MAY be implemented
-by child classes, and if so, will be called by the internal Writable
-class methods only.
+by child classes, and if so, will be called by the internal Writable class
```

Comment:
@GeorgeSapkin Yes, please. We like to keep diffs as small as possible as long as there is no noticeable benefit of having such changes.